sass/components/_colors.sass

```diff
@@ -0,0 +1,67 @@
+
+.color
+    display: inline-block
+    margin: 10px
+    text-align: center
+    max-width: 80px
+
+    .color-name
+        font:
+            size: 16px
+            weight: bold
+        text-transform: capitalize
+
+    .hexcode
+        font:
+            size: 13px
+
+.color-block
+    display: block
+    width: 80px
+    height: 80px
+    border-radius: 100%
+
+    .grey-1 &
+        background: $grey-1
+    .grey-2 &
+        background: $grey-2
+    .grey-3 &
+        background: $grey-3
+    .grey-4 &
+        background: $grey-4
+    .grey-5 &
+        background: $grey-5
+    .grey-6 &
+        background: $grey-6
+    .grey-7 &
+        background: $grey-7
+    .grey-8 &
+        background: $grey-8
+    .grey-9 &
+        background: $grey-9
+    .grey-10 &
+        background: $grey-10
+    .grey-11 &
+        background: $grey-11
+
+    // Example colors
+    // Using these color names as they are unlikely to be used in any of
+    // our projects and hopefully won't cause conflicts
+    .color1 &
+        background: #206ba4
```

Comment:
It would be really nice to somehow tie these colours to the ones declared in the gruntfile. I'm not sure exactly how we do that? Maybe generate a colours Sass file with grunt that we can import?